pymongo/encryption_options.py

```diff
@@ -166,6 +190,8 @@ def __init__(
                 "install a compatible version with: "
                 "python -m pip install 'pymongo[encryption]'"
             )
+        self._encrypted_fields_map = encrypted_fields_map
```

Comment:
It would be good to call validate_is_mapping here.

Reply:
Done.